example/radar/radar_scan_precip.py

```diff
@@ -87,10 +82,13 @@ def radar_scan_precip():
     files = [item.data for item in results]
 
     # Decode data using wradlib.
-    data = wrl.io.open_odim(files)
+    data = []
+    for file in files:
+        data.append(xradar.io.open_odim_datatree(file).to_dataset())
+    data = xarray.concat(data, dim="time")
```

Comment:
@kmuehlbauer it seems with xradar we are not able anymore to put in a list of files but should parse every single file by one and then stick them together afterwards, right?